### DISCUSSION TITLE
add instructions for normal people

### DISCUSSION THREAD

**Discussion by Gleboros** [cyberofficial/Synthalingua#134] [General]
It looks cool, but it didn't explain how to use it to reasonable people (not programmers and other types of assholes). In the "use" section, there are simply commands written that: where?, why?, where?, why?. What else is written in Elvish. Why don't the developers from GitHub understand that 99 percent of people on this site just download assets and use the application. As I understand it, there is no sensible interface here? Well, even if there is, did you show it? Explain it? You didn't explain anything.

**Comment by cyberofficial** [comment]
Hi! Thanks for taking an interest in the application and providing your feedback. I appreciate your honesty and want to assure you that I'm constantly working to make Synthalingua more user-friendly for everyone, regardless of their technical background.

I acknowledge that the README might be overwhelming for users who are not familiar with command-line interfaces. While the project initially started as a command-line tool, I've been actively developing a GUI (Graphical User Interface) to make it more accessible. If you read the Readme, you are greeted with "You can grab the GUI from the Releases section on github. Grab the portable version on itch!" in big bold letters you can not miss. 

**Here's how you can find and use the GUI version:**

1. **Go to the Releases section on GitHub:**  As mentioned at the top of the README, you can find the GUI version in the [Releases section](https://github.com/cyberofficial/Synthalingua/releases) of the GitHub repository. 
2. **Download the portable version:** On the Releases page, Click the Itch.io Image and look for the download button, the portable version is for Windows 10 x64 and newer. It's usually the largest file (around 3-4 GB) and has "Portable" in its name. The program is free and requires no payment, you can click the "No thanks take me to downloads" link or something similar to it. There is a no thanks button just click that. I use itch.io to allow users the option to use the itch.io launcher for auto updates and patches for easier management instead of downloading a 3gig file everytime.
3. **Run the executable:** Once downloaded, extract the files from the archive and run the executable file (SynthalinguaGUI.exe). This will launch the GUI version of Synthalingua. If you are downloading using the itch.io launcher then it'll ask what binary(exe) to launch and choose the GUI exe.

**Using the GUI:**

The GUI provides a more intuitive way to interact with Synthalingua. You can head over to the wiki page https://github.com/cyberofficial/Synthalingua/wiki/Graphical-User-Interface-‐-Information on how to properly use the GUI. If you still do not understand, please do reach out, I'm more than willing help out and guide you. 


**Regarding the "Elvish" commands:**

The commands you referred to are indeed for using Synthalingua through the command line. While they might seem complex at first, they offer more flexibility and customization for advanced users. I'm working on improving the documentation to make these commands easier to understand. These commands offer more flexibility for more advanced users that might be using linux or just prefer command line based programs/instructions and used for servers and alike. 

**My Goal:**

My goal is to make Synthalingua accessible and enjoyable for everyone. I'm continuously working on improving the GUI, adding features, and enhancing the documentation to achieve this. I encourage you to try the GUI version and provide any feedback you have. Your input is invaluable in helping me make Synthalingua better for all users. 

**Future Plans:**

I'm also planning to create video tutorials and guides that demonstrate how to use both the GUI and the command-line interface. This will hopefully bridge the gap and make Synthalingua more approachable for everyone. Currently on the itch.io page there is a youtube video that demonstrates the use of the gui application to an extent, though it is quite dated, the use of the gui didn't change, there might be a few new options/features, but overall is same. The wiki page is the most updated version of the information.

Thank you again for your feedback. I'm committed to making Synthalingua a tool that everyone can use and benefit from.
